examples/xlink_usb_client.cpp

```diff
@@ -0,0 +1,54 @@
+#include <cstdio>
+#include <string>
+#include <vector>
+#include <array>
+#include <thread>
+#include <stdexcept>
+#include <iostream>
+#include <cassert>
+#include <chrono>
+#include <thread>
+#include <algorithm>
+#include <cstring>
+#include <atomic>
+
+#include "XLink/XLink.h"
+#include "XLink/XLinkPublicDefines.h"
+#include "XLink/XLinkLog.h"
+
+// Common constants
+const uint8_t DUMMY_DATA[1024*128] = {};
+
+int main(int argc, char** argv) {
+    XLinkGlobalHandler_t gHandler;
+    XLinkInitialize(&gHandler);
+    
+    mvLogDefaultLevelSet(MVLOG_ERROR);
+
+    deviceDesc_t deviceDesc;
+    strcpy(deviceDesc.name, "usbdev");
+    deviceDesc.protocol = X_LINK_USB_EP;
```

Comment:
This should be USB_VSC

Reply:
USB_VSC will use EPs 81 and 1 to communicate, but those aren't the ones that are present in our gadget.
That's because in our gadget we create first the NCM function, then the XLink function, and after the ADB function.
I'll first modify the gadget and test it if it works.